docs/administration/Configuration.md

```diff
@@ -42,462 +42,2042 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
+
+#### user_function_dir
+
+- **Default**: ${STARROCKS_HOME}/lib/udf
+- **Unit**: N/A
+- **Description**: The directory used to store User-defined Functions (UDFs).
+
+#### small_file_dir
+
+- **Default**: ${STARROCKS_HOME}/lib/small_file
+- **Unit**: N/A
+- **Description**: The directory used to store the files downloaded by the file manager.
+
+#### sys_log_level
+
+- **Default**: INFO
+- **Unit**: N/A
+- **Description**: The severity levels into which system log entries are classified. Valid values: INFO, WARN, ERROR, and FATAL.
+
+#### sys_log_roll_mode
+
+- **Default**: SIZE-MB-1024
+- **Unit**: N/A
+- **Description**: The mode in which system logs are segmented into log rolls. Valid values include TIME-DAY, TIME-HOUR, and SIZE-MB-<size>. The default value indicates that logs are segmented into rolls, each of which is 1 GB.
+
+#### sys_log_roll_num
+
+- **Default**: 10
+- **Unit**: N/A
+- **Description**: The number of log rolls to reserve.
+
+#### sys_log_verbose_modules
+
+- **Default**: Empty string
+- **Unit**: N/A
+- **Description**: The module of the logs to be printed. For example, if you set this configuration item to OLAP, StarRocks only prints the logs of the OLAP module. Valid values are namespaces in BE, including starrocks, starrocks::debug, starrocks::fs, starrocks::io, starrocks::lake, starrocks::pipeline, starrocks::query_cache, starrocks::stream, and starrocks::workgroup.
+
+#### sys_log_verbose_level
+
+- **Default**: 10
+- **Unit**: N/A
+- **Description**: The level of the logs to be printed. This configuration item is used to control the output of logs initiated with VLOG in codes.
+
+#### log_buffer_level
+
+- **Default**: Empty string
+- **Unit**: N/A
+- **Description**: The strategy for flushing logs. The default value indicates that logs are buffered in memory. Valid values are -1 and 0. -1 indicates that logs are not buffered in memory.
+
+#### num_threads_per_core
+
+- **Default**: 3
+- **Unit**: N/A
+- **Description**: The number of threads started on each CPU core.
+
+#### compress_rowbatches
+
+- **Default**: TRUE
+- **Unit**: N/A
+- **Description**: A boolean value to control whether to compress the row batches in RPCs between BEs. TRUE indicates compressing the row batches, and FALSE indicates not compressing them.
+
+#### serialize_batch
+
+- **Default**: FALSE
+- **Unit**: N/A
+- **Description**: A boolean value to control whether to serialize the row batches in RPCs between BEs. TRUE indicates serializing the row batches, and FALSE indicates not serializing them.
+
+#### storage_root_path
+
+- **Default**: `${STARROCKS_HOME}/storage`
+- **Unit**: N/A
+- **Description**: The directory and medium of the storage volume.
+  - Multiple volumes are separated by semicolons (`;`).
+  - If the storage medium is SSD, add `medium:ssd` at the end of the directory.
+  - If the storage medium is HDD, add ,medium:hdd at the end of the directory.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/3568d558-3d4f-497d-8657-78d689a165c3)
Wrap ",medium:hdd" in code format too. Also, I think the comma (,) in ",medium:hdd" is a typo and should be removed?